examples/cxx-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Copyright 2016-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
Note for the future: It would be interesting to look into how well `add_wkt_library` interoperates with a `blt`-based build system.